### ISSUE TITLE
Create a new private Engineering Slack channel

### ISSUE THREAD

**Issue by damianavila** [2i2c-org/team-compass#725]
### Context

For historical reasons, we have used the `team-updates` channel as the _de facto_ engineering channel.
But now Engineering has grown enough to have its dedicated private channel, so we can decouple Eng-focused internal discussions from the team-updates channel.

### Proposal

1. [ ] Rename `@engineering` slack group to `@engineers` (otherwise we can't use the name in a new Slack channel
2. [ ] Create the new private `#engineering` channel.
3. [ ] Document the change in the team-compass.

I have discussed this proposal with several people already and I believe it is an improvement of the current situation so I will proceed with these steps as of now.

**Comment by choldgraf** [comment]
+1 from me, I think it makes sense given that there's a lot of engineering-specific chatter and process that's currently captured in `#team-updates`.

**Comment by choldgraf** [comment]
My assumption was "private to the 2i2c team" not "private only for engineering" but please correct he if I'm wrong @damianavila

**Comment by damianavila** [comment]
> My assumption was "private to the 2i2c team" not "private only for engineering" but please correct he if I'm wrong @damianavila

That's correct, this is not private to eng-only although is centered around Engineering (for the internal, not public-facing stuff). As I said above this is a decoupling of the team-updates channel into a more specific eng-y channel. Anyone from the 2i2c team is welcome to that channel (as with most of the other private channels in the 2i2c slack workspace) but I will add people on a by-user request basis (as with most of the other private channel in the 2i2c slack workspace). I want participation in that channel to be intentional in case you are not part of Engineering.

>Assuming that this new channel is not intended to be 'private' to engineering only, could you also add @choldgraf , @colliand, and @jmunroe ?

Because of the intentionality matter I raised, I will only add @jmunroe to the channel for now... if anyone else wants to be added, just let me know here and I will make that happen!

**Comment by colliand** [comment]
I wish to be added. Thanks in advance @damianavila!

**Comment by damianavila** [comment]
Both has been added.
